tests/src/Interop/PInvoke/SizeParamIndex/PInvoke/PassingByOut/PInvokePassingByOutNative.cpp

```diff
@@ -0,0 +1,97 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// PInvokePassingByOutNative.cpp : Defines the entry point for the DLL application.
+//
+#include <xplatform.h>
+#include "../helper.h"
```

Comment:
@AaronRobinsonMSFT @luqunl Do you have any suggestion about how to include files with relative path like this?
Is this OK?

Reply:
If this is indeed the location of the header, this is fine. Are you concerned about something specific?

Reply:
@AaronRobinsonMSFT There will be compile warnings about using the relative path "../" to include a file. Just want to make sure if it's fine to use the relative path here

Reply:
ugh.... What is the warning? You might want to try `..\helper.cpp`. If that is still a warning please add `..` to the include path in the `CMakeLists.txt` for the relevant projects.

Reply:
The warning is:
`warning C4464: relative include path contains '..'`
I will try to include in the CMakeLists.txt to see.

Reply:
This is fixed by adding `include_directories("..")` to CMakeList.txt and then `#include "helper.h"`